app/controllers/numismatic_citations_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+class NumismaticCitationsController < BaseResourceController
+  self.change_set_class = DynamicChangeSet
+  self.resource_class = NumismaticCitation
+  self.change_set_persister = ::ChangeSetPersister.new(
+    metadata_adapter: Valkyrie::MetadataAdapter.find(:indexing_persister),
+    storage_adapter: Valkyrie.config.storage_adapter
+  )
+  before_action :load_numismatic_references, only: [:new, :edit]
+
+  def create
+    # Save append_id and then remove from resource params
```

Comment:
can you add why to this comment?

Reply:
you're using the append_id list to get the parents to attach to?

Reply:
Yes. The append_id param is used to retrieve the parent numismatic issue or coin. The citation id is then inserted into the parent `numismatic_citation_ids` array.